package.json

```diff
@@ -1,20 +1,86 @@
 {
-  "name": "commercetools-nodejs-boilerplate",
+  "name": "@commercetools/nodejs-boilerplate",
   "version": "1.0.0-beta.1",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 0"
+  "description": "Generic Boilerplate for bootstrapping commercetools applications",
+  "private": true,
+  "keywords": [
+    "commercetools",
+    "nodejs",
+    "boilerplate",
+    "starter"
+  ],
+  "homepage": "https://github.com/commercetools/nodejs-boilerplate#readme",
+  "bugs": {
+    "url": "https://github.com/commercetools/nodejs-boilerplate/issues"
   },
+  "license": "MIT",
+  "contributors": [
+    "Abimbola Idowu <abimbola.idowu@commercetools.de>",
+    "Nicola Molinari <nicola.molinari@commercetools.de>",
+    "Philipp Sporrer <philipp.sporrer@commercetools.de>",
+    "Selwyn Versteeg <selwyn.versteeg@commercetools.de>"
+  ],
+  "main": "./lib/main",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/commercetools/nodejs-boilerplate.git"
+    "url": "https://github.com/commercetools/nodejs-boilerplate.git"
   },
-  "keywords": [],
-  "author": "Abimbola Idowu <abimbola2010@gmail.com> (https://github.com/hisabimbola)",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/commercetools/nodejs-boilerplate/issues"
+  "scripts": {
+    "start": "npm run test:watch",
+    "test": "cross-env NODE_ENV=test NODE_PATH=./src babel-node test/*.spec.js test/*/**spec.js | tap-spec",
+    "test:watch": "watch 'npm test' src test -d",
+    "build": "npm run clean && babel src -d lib",
+    "clean": "rimraf lib/*",
+    "commit": "git-cz",
+    "codecov": "cat ./coverage/lcov.info | codecov",
+    "coverage": "npm run lint && npm run clean && cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text npm test",
+    "coverage:html": "npm run lint && cross-env NODE_ENV=test nyc --reporter=html npm test",
+    "lint": "./scripts/lint src test"
+  },
+  "config": {
+    "commitizen": {
+      "path": "cz-conventional-changelog"
+    },
+    "ghooks": {
+      "commit-msg": "validate-commit-msg",
+      "pre-commit": "./scripts/lint-staged-files && npm test"
+    }
+  },
+  "eslintConfig": {
```

Comment:
This is not related to Babel. In the tests, we set the `NODE_PATH` variable which lets us import from `src` instead of having to reference it all the time (like `../../../src/...`).
There is, however, an eslint plugin called `eslint-plugin-import` which also resolves your import statements and checks whether they are valid. Thus we also need to tell this plugin, that we can also import/resolve from `src` directly.